source/agora/consensus/protocol/Data.d

```diff
@@ -16,6 +16,7 @@ module agora.consensus.protocol.Data;
 import agora.consensus.data.Enrollment;
 import agora.consensus.data.Transaction;
 import agora.crypto.Hash;
+import agora.crypto.Serializer;
```

Comment:
Why the move ?

Reply:
No reason. Moved back to unittest scope..